### DISCUSSION TITLE
Problem migrating from 0.27.5 to 1.0.0

### DISCUSSION THREAD

**Discussion by phyx3r** [botfront/botfront#925] [Q&A] [answered]
Hi to everyone! I'm very anxious to test the 1.0.0 release of Botfront. I've followed the migration guide but when I log in to the new dashboard I'm redirected to a 404 page. No idea how to debug the problem, I cannot read anything wrong in the logs. Thanks in advance!

**Comment by MatthieuJnon** [comment, marked as answer]
Have you started a brand new project with Botfront 1 ? Did you create your initial admin user and ended up on a 404 right after that? 

For clarifcation, you can't migrate the db from <1.0.0 to >= 1.0.0, you have to start a new project and import the exported files from 0.27.5 there

**Reply by phyx3r** [reply]
Hi @MatthieuJnon thanks for the reply. I've misunderstood the migration guide. I thought that I have to copy the db folder prior to import yaml configuration files.

Ok, I'm now starting a brand new BF1 project and trying to import the exported files but I cannot import any of the stories .md files with the following error:

user_inputs.md
Dialogue fragments in this file could not be parsed by Rasa at http://rasa-service.bf-libraway.

**Reply by MatthieuJnon** [reply]
can you share the user_inputs.md file ?

**Reply by phyx3r** [reply]
I'm sharing the chat_interactions.md file that is shorter than the user_inputs.md file. I cannot import anyone of them.

```
# Chat Interactions

## Start
* user_chitchat OR start
  - action_get_user_info
> Start__branches
## Start__Logged in
> Start__branches
  - slot{"is_logged":true}
  - utter_4ZE6gphm2
> checkpoint_0
## Start__Not logged in
> Start__branches
  - slot{"is_logged":false}
  - utter_ibvCqWGjG
  - utter_wOcrUKC21
> checkpoint_4
## Add note
* interaction_add_note
  - utter_0zQJt5Vyd
 ```

**Reply by phyx3r** [reply]
@MatthieuJnon I think that the problem may be the same of https://github.com/botfront/botfront/discussions/926 because currently the Botfront interface tells me that `Rasa instance not reachable`.
